extension/test/integration/config-loader.spec.js

```diff
@@ -0,0 +1,50 @@
+const fs = require('fs')
```

Comment:
This test looks more like a unit test.

Reply:
And it's better to put it in the `config.spec.js` unit test -  you don't really test the config-loader here.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/870/commits/db26ff3f7bb6f3e219deec4f91d2294e71b147d2